lib/Runtime/Types/ES5ArrayTypeHandler.cpp

```diff
@@ -271,25 +272,23 @@ namespace Js
             doneConversion = true;
 #endif
         }
-        __finally
+        catch(...)
```

Comment:
Is this commit slightly stale or is there a reason you decided to go with catch(...) instead of catch (Js::ExceptionBase)?